helm/prometheus-agent/charts/prometheus-agent/templates/prometheus.yaml

```diff
@@ -8,6 +8,18 @@ metadata:
   name: {{ include "name" . }}
   namespace: {{ .Release.Namespace }}
 spec:
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 100
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
```

Comment:
Should that not be based on the hostname instead?

Reply:
yes, my bad